doc/dev/index.rst

```diff
@@ -1021,6 +1050,53 @@ drive::
 
     $ ceph-workbench ceph-qa-suite --suite dummy
 
+Your first run of `ceph-workbench ceph-qa-suite`_ on a given tenant will
+take a long time to complete because it downloads a VM image. The images
+are cached in FIXME, so they are only downloaded once.
+
+Although ``dummy`` suite does not run any tests, in all other respects it
+behaves just like a teuthology suite and produces some of the same
+artifacts (see `Artifacts produced by teuthology-suite`_).
+
+You can tell whether the ``dummy`` suite completed successfully by FIXME.
```

Comment:
Unfortunately the `dummy` suite shows up in Pulpito as 1 dead job, so it doesn't really succeed.